apps/taxes/src/modules/taxjar/taxjar-validation.service.ts

```diff
@@ -0,0 +1,39 @@
+import { TaxjarError } from "taxjar/dist/util/types";
+import { taxJarAddressFactory } from "./address-factory";
+import { TaxJarClient } from "./taxjar-client";
+import { TaxJarConfig } from "./taxjar-config";
+
+export class TaxJarValidationService {
+  constructor() {}
+  private resolveTaxJarValidationError(error: unknown): Error {
+    if (!(error instanceof TaxjarError)) {
+      return new Error("Unknown error while validating TaxJar configuration.");
```

Comment:
if we can't parse error we should probably
1. send it to Sentry (or ensures its sent)
2. Log this error so we can find it in logs

If error is not parsed, its "our problem" we need to solve

Reply:
1. I have to make sure it's sent, but it should be.
2. It's logged in the `webhook-response` class.